### PR TITLE
Made ucs-restart.bat pretty useful!

### PR DESCRIPTION
**Pull request by smartclash**
I think this also should be took into consideration.

### CODE REVIEW

File: Ultrapowa Clash Server/bin/Debug/tools/ucs-restart.bat [hunk 1]
```diff
@@ -1,4 +1,6 @@
-echo off
+@echo off
```
Comment (smartclash) [inline Ultrapowa Clash Server/bin/Debug/tools/ucs-restart.bat]: aidid missed this thing `@`

File: Ultrapowa Clash Server/bin/Debug/tools/ucs-restart.bat [hunk 2]
```diff
@@ -1,4 +1,6 @@
...
+title UCS restarter
```
Comment (smartclash) [inline Ultrapowa Clash Server/bin/Debug/tools/ucs-restart.bat]: Added a title

File: Ultrapowa Clash Server/bin/Debug/tools/ucs-restart.bat [hunk 3]
```diff
@@ -7,9 +9,17 @@ echo 	888     888 888            ""Y88b.
...
 echo         "Y88888P"   "Y8888P"   "Y8888P"
-echo
-color 0B   
-echo UCS-Restart-Op v0.1 by Aidid   
+echo.
+echo.
+color F4  
+echo UCS restarter created for 0.6.5.0 
+echo A programme to restart UCS and prevent Laging. 
+echo Leave this programme to run at the backgorund while UCS is running.
```
Comment (smartclash) [inline Ultrapowa Clash Server/bin/Debug/tools/ucs-restart.bat]: Gave some information

File: Ultrapowa Clash Server/bin/Debug/tools/ucs-restart.bat [hunk 4]
```diff
@@ -7,9 +9,17 @@ echo 	888     888 888            ""Y88b.
 echo 	888     888 888    888       ""888 
 echo 	Y88b. .d88P Y88b  d88P Y88b  d88P 
 echo         "Y88888P"   "Y8888P"   "Y8888P"
-echo
-color 0B   
-echo UCS-Restart-Op v0.1 by Aidid
```
Comment (smartclash) [inline Ultrapowa Clash Server/bin/Debug/tools/ucs-restart.bat]: Useless credit

File: Ultrapowa Clash Server/bin/Debug/tools/ucs-restart.bat [hunk 5]
```diff
@@ -20,5 +30,6 @@ echo Success!
 echo.   
 echo Your UCS is now starting. . .   
 timeout /t 15
-start ucs.exe   
-exit 
+start ucs.exe 
+cls
+goto :x
```
Comment (smartclash) [inline Ultrapowa Clash Server/bin/Debug/tools/ucs-restart.bat]: Made a loop to continually reboot the server

**Comment by iSwuerfel** [comment]
I think it would be better if there would be a Built-in restarter (maybe like a command..?) because a bat restarter doesn't save the data before the shutdown.

**Comment by ADeltaX** [comment]
Already implemented on GUI version. I will add here for "completion".

**Comment by aididhaiqal** [comment]
UCS-Restart bat is not for auto restart.Its for the /restart command P/S This bat gonna be deleted soon as i got a new way to restart ucs after saving

**Comment by smartclash** [comment]
@iSwuerfel @ADeltaX @aididhaiqal Yup, even i did have the same thought. A bat file is not effective and will not save users before exiting.And is the source code of 0.6.3.1 available? And in-built restart command will be usefull.

**Pull request closed without merging**